### PR TITLE
LRQA-78055 | master

### PR DESCRIPTION
**Pull request by GergoMathe**
[LRQA-78055](https://liferay.atlassian.net/browse/LRQA-78055)

### CODE REVIEW

**Comment by GergoMathe** [comment]
ci:help

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:ldap:nocompile

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:ldap:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 1]
```diff
@@ -500,6 +500,60 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-78055. Liferay Admin user, for the virtual instance, is not able to login after the 'Required' box in LDAP configurations is turned on. ."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:503]: Please update the descirption, according the ticket, what we test here is the happy path, not the negative aka original issue.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 2]
```diff
@@ -500,6 +500,60 @@ definition {
...
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+
+			LDAP.addLDAPServerAndTestConnection(
+				baseURL = "http://www.able.com:8080",
+				ldapServerName = "ApacheDS Server");
+		}
+
+		task ("When: 'Enabled' and 'Required' settings are turned on in LDAP settings") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:535]: Please mention the settings are enabled on both default and virtual instances

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
See my review comments

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase (hunk 1 above)
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase]: Fixed.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 3]
```diff
@@ -500,6 +500,60 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-78055. Liferay Admin user, for the virtual instance, is not able to login after the 'Required' box in LDAP configurations is turned on. ."
+	@priority = 4
+	test LoginWithRequiredBoxTurnedOn {
```
[the remark this answers is not in the document]
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase]: Fixed.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase (hunk 2 above)
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase]: Fixed.

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:test:ldap:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**